### ISSUE TITLE
Login with Fig Failure/Infinite Loop

### ISSUE THREAD

**Issue by jwebbstevens** [withfig/fig#1134]
### Sanity checks

- [X] I have searched [github.com/withfig/fig/issues](https://github.com/withfig/fig/issues?q=) and there are no duplicates of my issue
- [X] I have run `fig doctor` in the affected terminal session
- [X] I have typed 'git ' and included a screenshot of the Debugger (click the Fig icon in the macOS menu bar)

### Issue Details

Issue:

Cannot login to MacOS UI and access Fig Settings, blocked by login screen.

### Description:
1. Navigate to "Settings"
2. Open open presented with a Login screen.
3. Enter Email
4. 6 digit code sent to Email
5. Enter Code
6. Click Verify Email
7. Repeats from 1

*running Fig Doctor corrects the issue but no issues found

Fig Version 1.0.56 (B414)
MasOS Version 12.3 Monterey 

### Environment

```bash
- Fig version: Version 1.0.56 (B414) [U.S.]
  - Bundle path: /Applications/Fig.app
  - Autocomplete: true
  - Settings.json: true
  - Accessibility: true
  - Number of specs: 199
  - Symlinked dotfiles: false
  - Only insert on tab: false
  - Keybindings path: 
  - Installation Script: true
  - PseudoTerminal Path: hidden
  - SecureKeyboardInput: false
  - SecureKeyboardProcess: <none>
## Hardware Info:
  - Model Name: MacBook Pro
  - Model Identifier: MacBookPro15,2
  - Chip: 
  - Cores: 4
  - Memory: 16 GB
## OS Info:
  - macOS 12.3.0 (21E230)
## Environment:
  - User Shell: /bin/zsh
  - Current Directory: /Users/hidden
  - CLI Installed: true
  - Executable Location: /usr/local/bin/fig
  - Current Window ID: 4749/% (com.apple.Terminal)
  - Active Process: zsh (42800) - /dev/ttys002
  - Environment Variables:
    - TERM=xterm-256color
    - TERM_SESSION_ID=82C95D61-7A4A-49E4-BC84-5E2F6BB79AE4
    - PATH= (hidden for security)
    - FIG_INTEGRATION_VERSION=8
    - FIG_TERM=1
    - FIG_TERM_VERSION=4.3.0
## Integrations:
  - SSH: false
  - TMUX: false
  - iTerm: application is not present.
  - Hyper: application is not present.
  - Visual Studio Code: application is not present.
  - Docker: fals
```

**Comment by brendanfalk** [comment]
So confirming: you ran `fig doctor` and the issue was fixed?

Seems like we need better error messages in our `fig` app

Let us know

**Comment by mschrage** [comment]
This may occur if you are using a corporate VPN. Please share details about your networking setup if this is the case.

**Comment by eisenhand89** [comment]
using `fig login` in the terminal and entering the verification code there managed me to get me logged in.

**Comment by jwebbstevens** [comment]
> This may occur if you are using a corporate VPN. Please share details about your networking setup if this is the case.

Not using a corporate VPN.  

Network is pretty run of the mill.  Using a PiHole though connected to DNSSEC via Cloudflare on IPv4 (no IPv6 active) (I'm ad-less at home), looking through the logs I don't see anything unusual related to Fig but I can't replicated the endless loop, which happened right after I upgraded to Version 1.0.56 (B414) 

When I check the UI the Account/User Profile section is 'coming soon' but only after using `fig doctor`


<img width="488" alt="Screen Shot 2022-03-24 at 7 01 33 PM" src="https://user-images.githubusercontent.com/7277574/160024148-0b2d5b68-e559-4d22-8b29-5996ff6d089e.png">





> using `fig login` in the terminal and entering the verification code there managed me to get me logged in.

I think I restarted the application, used `fig doctor` and then restarted again and was able to access settings in the UI.

I tried this just now `fig login` got the code emailed to me entered it. But it didn't do anything, I guess that's part of the 'coming soon' part.

Not really sure what was causing the endless loop thing.  Not sure if pihole/dns is involved. Is your service hitting anything that might be considered an ad and on a general blocklist?  I don't use any blocklists other than the default.

**Comment by jwebbstevens** [comment]
> So confirming: you ran `fig doctor` and the issue was fixed?
> 
> Seems like we need better error messages in our `fig` app
> 
> Let us know

yep, appears to have fixed it.  i restarted it as well. not sure if that matters.

**Comment by mschrage** [comment]
@jwebbstevens @eisenhand89 we've just pushed a hotfix to address this! 

If you are still running into this issue, make sure to close the Fig window and reopen it.

(You may need to login one last time, but after that everything should work normally).
